src/Command/WorkerCommand.php

```diff
@@ -56,7 +56,7 @@ protected function configure()
                 '',
                 InputOption::VALUE_OPTIONAL,
                 "Gracefully stop one worker ($whenHints)",
-                'soon'
+                false // default value is used when option not given
```

Comment:
hint I missed before is here: https://symfony.com/doc/current/console/input.html

Reply:
:+1: